src/main/java/uk/gov/digital/ho/hocs/casework/application/CacheWarmer.java

```diff
@@ -69,7 +67,7 @@ private void primeStandardLinesCache(){
                 this.infoClient.populateStandardLine(getStandardLineResponse.getTopicUUID(), getStandardLineResponse);
             }
         } catch(Exception e) {
-            log.warn("Failed to prime StandardLines. :" + e.toString(), value(EVENT, CACHE_PRIME_FAILED));
+            log.warn("Failed to prime StandardLines. : {}", e.toString(), value(EVENT, CACHE_PRIME_FAILED));
```

Comment:
These should look as the exception value rather than as an inline string. 
e.g. log.warn("Failed to prime StandardLines.", value(EVENT, CACHE_PRIME_FAILED), value(EXCEPTION, value(e.toString());

Reply:
I agree, ill do this a separate piece for the whole service